int32 camera_id
bool activate
---
bool success
string message